src/components/Dashboard/Form/Applicants/Applicant.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../../../scss/_mixins.scss';
+.Applicant{
+  @include flex(wrap);
+  width: 650px ;
+}
+button.developedDiv {
+  background-color: $orange;
+  border: none;
+  border-radius: 50%;
+  color: $white;
+  cursor: pointer;
+  font-size: $font-lg;
+  font-weight: $bold;
+  margin: 30px 0;
+  outline: none;
```

Comment:
just a tip for the future, outline is good for accessibility, while setting it to none you remove it, then a solution is quite easy, you can add `outline-color` and set it to transparent also add `outline-style` and set it to none

Reply:
thank you :) done